hack/go-mod-prepare.sh

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
-ROOTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+set -e
 
-cat > "${ROOTDIR}/go.mod" << EOF
+SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOTDIR="$(git -C "$SCRIPTDIR" rev-parse --show-toplevel)"
+
+set -x
+
+tee "${ROOTDIR}/go.mod" <<EOF
```

Comment:
Was there a specific reason to use `tee` for this? This also prints the `go.mod` on stdout (whereas the old one only wrote it to the file)

Reply:
The tee is intended to increase transparency (along with the `set -x`) for debugging/CI. It's a bit more noise, but I do think it makes the steps being run much more clear to the user.